### PR TITLE
Use subtitle resource for SAMSUNG clients if available

### PR DESCRIPTION
**Pull request by KarlStraussberger**
Remove unused method

Found this while researching on #1165 

There is still a bit of hardcoding for srt files which should be bypassed for explicit resources already.

### CODE REVIEW

File: test/core/test_upnp_xml.cc [hunk 1]
```diff
@@ -125,19 +125,6 @@ TEST_F(UpnpXmlTest, RenderObjectItem)
     EXPECT_STREQ(didl_lite_xml.c_str(), expectedXml.str().c_str());
 }
 
-TEST_F(UpnpXmlTest, CreatesSecCaptionInfoElement)
```
Comment (whyman) [inline test/core/test_upnp_xml.cc:128]: It would be nice to add some tests too

**Comment by whyman** [comment]
If we enabled the subs patterns:
``` 
"<filename>.srt",
"<filename>.ssa",
"<filename>.smi",
"<filename>.sub"
```
By default, then we could get rid of that munging entirely?

**Comment by KarlStraussberger** [comment]
> 
> 
> If we enabled the subs patterns:
> 
> ```
> "<filename>.srt",
> "<filename>.ssa",
> "<filename>.smi",
> "<filename>.sub"
> ```
> 
> By default, then we could get rid of that munging entirely?

I'd leave that change for some time after 1.7 (feel free to create an issue). The code is note completely clear to me and adding all sub patterns is a performance killer.

File: test/core/test_upnp_xml.cc (hunk 1 above)
Comment (KarlStraussberger) [reply test/core/test_upnp_xml.cc:128]: Added a combined test for albumart and subtitle

**Pull request merged**